src/css.rs

```diff
@@ -92,6 +92,14 @@ pub enum CssParseError<'a> {
     UnexpectedValue(CssParsingError<'a>),
 }
 
+impl_display!{ CssParseError<'a>, {
+    ParseError(e) => format!("Parse Error: {:?}", e),

```

Comment:
Use `{}` here, otherwise all your `impl Display` changes don't affect the final outcome.

Reply:
I did that because the underlying error is a `simplecss::CssSyntaxError` which does not implement Display..

Reply:
Okay, then leave it as is.